card/spec/models/card_spec.rb

```diff
@@ -83,14 +83,14 @@ def assert_stable card1
   end
 
   it 'should remove cards' do
-    forba = Card.create! :name=>"Forba"
-    torga = Card.create! :name=>"TorgA"
-    torgb = Card.create! :name=>"TorgB"
-    torgc = Card.create! :name=>"TorgC"
+    forba = Card.create! name: "Forba"
+    torga = Card.create! name: "TorgA"
+    torgb = Card.create! name: "TorgB"
+    torgc = Card.create! name: "TorgC"
 
-    forba_torga = Card.create! :name=>"Forba+TorgA";
-    torgb_forba = Card.create! :name=>"TorgB+Forba";
-    forba_torga_torgc = Card.create! :name=>"Forba+TorgA+TorgC";
+    forba_torga = Card.create! name: "Forba+TorgA";
```

Comment:
Useless assignment to variable - `forba_torga`. Did you mean `forba_torga_torgc`?<br>Do not use semicolons to terminate expressions.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.